phpunit.xml.dist

```diff
@@ -30,4 +30,8 @@
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
+
```

Comment:
There are a lot more of them @OskarStark (8 to be precise). And you cant rewrite them correctly. If your php.ini uses precision 200, you will get other results than with precision 5. So the correct way in my opinion is to write your tests with the knowledge of the php.ini precision (thats what I did with the xml change). If you're not comfortable with precision "8" we could also use "100" and then round the values before comparing in the unit test.